helm/charts/determined/values.yaml

```diff
@@ -143,8 +142,42 @@ taskContainerDefaults:
   # (CPU notebooks, TensorBoards, zero-slot commands). If a pod spec is defined for an individual
   # task, that pod spec will replace the default one that is defined here. See
   # https://docs:determined.ai/latest/topic-guides/custom-pod-specs.html for more details.
-  # cpuPodSpec:
-  # gpuPodSpec:
+  cpuPodSpec:
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      labels:
+        customLabel: cw-cpu-determined-pod
+    spec:
+      containers:
+        - name: determined-cpu-container
+          resources:
+            requests:
+              memory: {{.Values.resources.cpu}}
+              cpu: {{.Values.resources.memory}}
+            limits:
+              memory: {{.Values.resources.cpu}}
+              cpu: {{.Values.resources.memory}}
+  gpu_pod_spec:
+    apiVersion: v1
```

Comment:
Do you really need to define the apiVersion stuff in here? That's not there by default?

Reply:
Removed it.